tareas/Seal/Tarea1.1.jl

```diff
@@ -0,0 +1,91 @@
+struct Dual{T <: Real} <: Real   ##Para derivadas multiples,eso creo
```

Comment:
```suggestion
module DifAutom

export Dual, dual, fun, der

struct Dual{T <: Real} <: Real   ##Para derivadas multiples,eso creo
```

Agrega estas dos lineas arriba